README.md

```diff
@@ -132,6 +136,31 @@ params:
   - compute.googleapis.com/instance/disk
 ```
 
+### What to know about Aggregating DELTA Metrics
+
+Treating DELTA Metrics as a gauge produces data which is wildly inaccurate/not very useful (see https://github.com/prometheus-community/stackdriver_exporter/issues/116). However, aggregating the DELTA metrics overtime is not a perfect solution and is intended to produce data which mirrors GCP's data as close as possible. 
+
+The biggest challenge to producing a correct result is that a counter for prometheus does not start at 0, it starts at the first value which is exported. This can cause inconsistencies when the exporter first starts and for slow moving metrics which are described below.
```

Comment:
Just to note, we have an OpenMetrics solution for the starting-at-zero problem. But we need Prometheus support for OpenMetrics created timestamps.

We need this for a bunch of reasons, so I've added it to the Promethus dev summit agenda so we can talk about it again.